security/fma-operator-fee.md

```diff
@@ -122,17 +163,45 @@ Below are references for this project:
     - If the bug is located in op-node, a new version must be deployed.
     - If the bug is located in the `L1Block` contract, the contract must be upgraded to fix the bug.
 
+### FM9: Overflow/Underflow of operator fee calculation
+- **Description:** If the operator fee calculation is vulnerable to overflows or underflows then it could be possible to manipulate the calculation through, say, a deliberately large gasUsed in order to trigger that overflow or underflow and charge a negative fee. 
+- **Risk Assessment:** high severity / low likelihood
+- **Mitigations:**
+  - The calculation cannot overlow or underflow. See context below:
+
+    > `gas` is a uint64, so `(gasUsed * operatorFeeScalar / 1e6) + operatorFeeConstant` can be at most `(u64.max * u32.max / 1e6) + u64.max ~= 7.924660923989131e+22`, which is an int of bit length 77 and fits comfortably within a uint256 variable allocation.
+
+- **Detection:**
+  - NAT fuzz testing (TODO)
```

Comment:
I would prefer to remove this. It doesn't make sense to fuzz something that is mathematically impossible. If anything, we should add a test that just uses the intN.max values for all fields to show that even with the highest possible input parameters, it doesn't overflow.